buku

```diff
@@ -3375,6 +3383,34 @@ def convert_bookmark_set(
             count += 1
 
         out += '</xbel>'
+    elif export_type == 'rss':
+        timestamp = datetime.datetime.now().isoformat()
+        out = (
+            '<feed xmlns="http://www.w3.org/2005/Atom">\n'
+            '<title>Bookmarks</title>\n'
+            '<generator uri="https://github.com/jarun/buku">buku</generator>\n'
+            '<updated>%s</updated>\n' % timestamp
+        )
+
+        for row in resultset:
+            out += '<entry>\n'
+            out += '        <title>' + title(row) + '</title>\n'
+            out += '        <link href="%s" rel="alternate" type="text/html"/>' % html.escape(row.url)\
+                .encode('ascii', 'xmlcharrefreplace')\
+                .decode('utf-8')
```

Comment:
…Incidentally, instead of splitting the argument into multiple lines, I'd suggest calculating it separately:
```py
_url = html.escape(row.url).encode('ascii', 'xmlcharrefreplace').decode('utf-8')
out += '        <link href="%s" rel="alternate" type="text/html"/>' % _url
```